lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -719,6 +689,25 @@ if Enum.all?(
       }
     end
 
+    defp filter_placements({Pad.ref(_type, id), _placement}, track),
+      do: id == track.id or id == {:blank, track.id}
+
+    defp update_layout_action(_track, _video_layout, %{mixer_config: nil} = state),
+      do: {[], state}
+
+    defp update_layout_action(%{type: :audio}, _video_layout, state), do: {[], state}
+
+    defp update_layout_action(track, :removed, state) do
+      {placements, video_layout} =
+        state.mixer_config.video.layout_module.track_removed(state.video_layout, track)
+
+      update_action = [forward: {:compositor, {:update_placement, placements}}]
+      {update_action, %{state | video_layout: video_layout}}
```

Comment:
What do you think about this change?
```suggestion
      state = %{state | video_layout: video_layout}
      update_layout_action(track, placements, state)
```

Reply:
looks really nice